dask_ml/preprocessing/data.py

```diff
@@ -210,3 +210,22 @@ def _transform_col(self, X_col, quantiles, inverse):
             X_col = da.clip(X_col, clip_min, clip_max)
 
         return X_col
+
+
+class RobustScaler(skdata.RobustScaler):
+
+    def _check_array(self, X, copy):
+        return X
```

Comment:
FYI: adding a `utils.validation.check_array` that works on dask arrays is near the top of my TODO list.